### DISCUSSION TITLE
feature request: add optional title to the card

### DISCUSSION THREAD

**Discussion by commentsonly** [kinghat/tabbed-card#29] [Feature Requests]
Played some days with the sub-view thingy ..... and dropped it now, since if too much information is available using tabs is the more common way I got used to. Wonder how HA worked without that one.

Perhaps you could think about a card-title? above the tabs? Would make something like this obsolete:

> type: vertical-stack
title: Livingroom
cards:
  \- type: custom:tabbed-card

apart from that ... it easily climbed up the ranking of my must-have-custom-cards

**Comment by kinghat** [comment]
> Played some days with the sub-view thingy ..... and dropped it now, since if too much information is available using tabs is the more common way I got used to. Wonder how HA worked without that one.

were you the person i had the thread with on discord?

> Perhaps you could think about a card-title? above the tabs? Would make something like this obsolete:
> 
> > type: vertical-stack
> > title: Livingroom
> > cards:
> > - type: custom:tabbed-card

i was thinking about this as well. could you open a feature discussion under [ideas](https://github.com/kinghat/tabbed-card/discussions/categories/ideas) so it doesnt get lost?

> apart from that ... it easily climbed up the ranking of my must-have-custom-cards

:pray:

**Comment by ASNNetworks** [comment]
Please make the title _optional_, since I don't use a title because I have my own titels in place thrhoughout my UI.

**Reply by kinghat** [reply]
it would be optional.
